lib/convenient_service/common/plugins/sets_display_value/middleware.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ConvenientService
+  module Common
+    module Plugins
+      module SetsDisplayValue
```

Comment:
Let's remove this plugin in favor of config.

Please, check an example with [RSpec](https://github.com/marian13/convenient_service/blob/main/lib/convenient_service/configs/standard.rb#L210).